content/articles/build-digital-clock-using-javascript/index.md

```diff
@@ -0,0 +1,311 @@
+ 
+
+title: How to build a digital clock with only HTML, CSS, and JavaScript
+
+### Introduction
+
+The web made almost everything possible. The emergence of JavaScript as the programming language of the web enables us to see our code live and interact with web content. This article seeks to use this important tool to build a digital clock that runs on the web with other web-based technologies such as HTML, CSS.
+
+### Prerequisites
+
+The reader should have basic knowledge of:
+
+- JavaScript
+
+- HTML
+
+- CSS 
+
+- Have a text editor of his or her choice, though I recommended vscode
+
+- Internet connectivity
+
+### Objectives
+
+At the end of this tutorial, readers should be able to:
+
+- Understand Javascript's built-in `setInterval` function
+
+- Know how to use the JavaScript Date class
+
+- Know how to create a function
+
+- Know how to invoke or call a function
+
+- Know what variable scope is
```

Comment:
You didn't seem to talk about variable scope anywhere here. Either add a section on variable scope or remove this.

Reply:
I did mention it in line 266 but I fail to explain what it means. I will add a section.